### DISCUSSION TITLE
WSA not working on a custom Windows 10 IoT Enterprise LTSC 21H2 ROM.

### DISCUSSION THREAD

**Discussion by Zeralyx** [MustardChef/WSABuilds#204] [Q&A]
Why do I get this specific error? I tried searching on google and on the forums here but im not an expert on coding and software so IDK what this error means OR how to fix it. Any help would  be appreciated!

![Screenshot_6](https://github.com/MustardChef/WSABuilds/assets/147331703/eaea7db0-587e-46f3-b70b-6f1e16e4f861)

**Comment by MustardChef** [comment]
@Zeralyx Try running:
```powershell
Import-Module -Name Appx -UseWindowsPowerShell
PowerShell.exe -ExecutionPolicy Bypass -File .\Install.ps1
```
in the same folder as all the WSA files.

**Reply by Zeralyx** [reply]
Hey, so I assumed you wanted me to right click and use PowerShell on the WSA folder with that code. I did that AND same error BUT I tried again, instead of Open here I tried Open here as administrator AND IT WORKED OMG. 

Is it safe now to close the PowerShell window? It says Installing MagiskonWSA... but nothing happens so im assuming its all done and working?:)
![Screenshot_7](https://github.com/MustardChef/WSABuilds/assets/147331703/13f850a2-6ffa-4a06-b869-39ddd73411ab)

**Reply by MustardChef** [reply]
Yes, you can close the Powershell Window.
